lib/main.js

```diff
@@ -1,7 +1,6 @@
 /*!
- * Copyright (c) 2020-2021 Digital Bazaar, Inc. All rights reserved.
+ * Copyright (c) 2021 Digital Bazaar, Inc. All rights reserved.
  */
-import 'bedrock-express';
 
-import './config.js';
-import './http.js';
+export {OAuth2Client} from './oauth2-client.js';
+export {get, insert} from './storage.js';
```

Comment:
So looking more closely at this, I think it's odd to export `get` and `insert` directly like this. We should either export the full storage API using the name `storage` or not export it at all.

Reply:
Updated.